Docs/articles/Kernel/Network.md

```diff
@@ -51,6 +51,26 @@ using(var xClient = new UdpClient(4242))
 
 ## TCP
 Like UDP, TCP has to create a client and call Connect() to specify the remote machine address before sending or receiving data.
+
+Server :
+```csharp
+using(var xServer = new TcpListener(4242))
+{
+    /** Start server **/
+    xServer.Start();
+    
+    /** Accept incoming TCP connection **/
+    var client = xServer.AcceptTcpClient(); //blocking
+    
+    /** Stop server **/
+    xServer.Stop();
```

Comment:
Stop is called via Dispose indeed but it's to show all possible method calls inside the class

Reply:
I think it would be better if you mention that you should either use a `using` block or call Stop explicitly. The current example is confusing and would encourage incorrect behavior.